src/main/kotlin/com/terraformation/backend/documentproducer/DocumentUpgradeCalculator.kt

```diff
@@ -210,8 +214,10 @@ class DocumentUpgradeCalculator(
               is SectionValueText -> sectionValue.value
               is SectionValueVariable -> {
                 val existingUsedVariableId = sectionValue.value.usedVariableId
+                // This needs to come from all variables, since variables now
```

Comment:
Super minor quibble: replace "now" with "can" since someone reading this comment down the road will probably only care about why the code currently behaves the way it does, not how it differs from what used to happen in the document producer.

Reply:
Excellent feedback, thanks